devtools/package.json

```diff
@@ -33,6 +33,6 @@
     "cross-env": "^5.1.3"
   },
   "scripts": {
-    "postinstall": "cross-env empathy install -a deps --ignore diff vis web-animations-js && cross-env tools/css-module-wrap ./node_modules/vis/dist/vis-timeline-graph2d.min.css"
+    "postinstall": "cross-env empathy install -a deps --ignore diff vis web-animations-js &> /dev/null && cross-env tools/css-module-wrap ./node_modules/vis/dist/vis-timeline-graph2d.min.css"
```

Comment:
Would `1>/dev/null` (assuming the error messages are on stdout) be a smaller scope, so that stderr messages would still be reported?

Does this work on Windows?

Reply:
(maybe @sjmiles knows about the second question - this looks like it would result on an error in Windows, but maybe `cross-env` is more magic than I realize)

Reply:
This fails on Windows. I poked around briefly but didn't find a way to do this with `cross-env`.